Resources/Prototypes/DeltaV/Loadouts/loadout_groups.yml

```diff
@@ -178,6 +178,31 @@
   loadouts:
   - ProsecutorNeck
 
+# Command
+## Admin Assistant
+
+- type: loadoutGroup
+  id: AdminAssistantJumpsuit
+  name: loadout-group-admin-assistant-jumpsuit
+  loadouts:
+  - AdminAssistantJumpsuit
+  - AdminAssistantJumpskirt
+
+- type: loadoutGroup
+  id: AdminAssistantOuterClothing
+  name: loadout-group-admin-assistant-outerclothing
+  minLimit: 0
+  loadouts:
+  - CaptainWintercoat
```

Comment:
pretty please change this out for something else. duravest would work fine for now until your or I get to spriting a dedicated coat